tests/eas/acceptance.py

```diff
@@ -240,141 +240,49 @@ class OffloadMigrationAndIdlePull(unittest.TestCase):
-
-        wload = RTA(
-            cls.env.target,
-            "offload_idle_pull",
-            calibration=cls.env.calibration())
-        wload.conf(kind="profile", params=cls.params)
-        cls.env.ftrace.start()
-        wload.run(
-            out_dir=cls.env.res_dir,
-            background=False)
-        cls.env.ftrace.stop()
-        trace = cls.env.ftrace.get_trace(cls.trace_file)
-
-    @classmethod
-    def get_offset(cls):
-        task_start_times = cls.a_assert.getStartTime().values()
-        return min([t['starttime'] for t in task_start_times])
+    conf_basename = "offload_idle_pull.config"
+    # m_assert = SchedAssert for migrators
+    # m_assert = for all
 
     @classmethod
-    def calculate_end_times(cls):
+    def calculate_end_times(cls, experiment):
 
         end_times = {}
-        for task in cls.params.keys():
-            sched_assert = SchedAssert(cls.trace, cls.env.topology,
+        for task in experiment['wload'].tasks.keys():
+            sched_assert = SchedAssert(experiment['out_dir'], cls.te.topology,
                                        execname=task)
             end_times[task] = sched_assert.getEndTime()
 
         return end_times
 
-    def get_migrator_activation_time(self):
-        start_times_dict = self.m_assert.getStartTime()
-        start_time = min(t['starttime'] for t in start_times_dict.itervalues())
-
-        return start_time + OFFLOAD_MIGRATION_MIGRATOR_DELAY
-
-    def test_first_cpu_early_starters(self):
-        """Offload Migration and Idle Pull: Test First CPU (Early Starters)"""
-
-        logging.info(
-            "Offload Migration and Idle Pull: Test First CPU (Early Starters)")
-        log_result(
-            self.e_assert.getFirstCpu(), self.log_fh)
-
-        self.assertTrue(
-            self.e_assert.assertFirstCpu(
-                self.env.target.bl.bigs,
-                rank=self.num_tasks),
-            msg="Not all the new 'early starter' tasks started on a big CPU")
-
-    def test_first_cpu_migrators(self):
-        "Offload Migration and Idle Pull: Test First CPU (Migrators)"
-
-        logging.info(
-            "Offload Migration and Idle Pull: Test First CPU (Migrators)")
-
-        log_result(
-            self.m_assert.getFirstCpu(), self.log_fh)
-
-        self.assertTrue(
-            self.m_assert.assertFirstCpu(
-                self.env.target.bl.bigs,
-                rank=self.num_tasks),
-            msg="Not all the new 'migrator' tasks started on a big CPU")
-
-    def test_big_cpus_fully_loaded(self):
+    @experiment_test
+    def test_big_cpus_fully_loaded(self, experiment, tasks):
         """Offload Migration and Idle Pull: Big cpus are fully loaded as long as there are tasks left to run in the system"""
-        num_big_cpus = len(self.env.target.bl.bigs)
+        num_big_cpus = len(self.te.target.bl.bigs)
 
-        end_times = sorted(self.end_times.values())
+        s_assert = self.get_multi_assert(experiment, "early")
```

Comment:
I'm not sure this can really be done since using LisaTest the `SchedMultiAssert`s need to be per-experiment and we don't have access to the experiment or tasks until the `test_` method. I'll have a think about it.